learn/framework/architecture.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Framework Architecture
+description: a Substrate node has two main components, the runtime, controlling the state transition of the blockchain, and the client, controlling everything else.
+---
+
+# Framework Architecture {: #framework-architecture } 
+
+## Introduction {: #introduction }
+
+Substrate is a software development kit (SDK) for building blockchains. This framework is the foundation and engine powering the Polkadot's relay chain, the parachains (such as the Tanssi network itself), and the ContainerChains deployed through Tanssi. 
+
+Written in the Rust Language and designed with a modular architecture, Substrate is extremely performant, flexible, and highly customizable, making it the best choice for developing blockchains.
+
+In this article, the architecture of a Substrate node is covered.
+
+## Architecture {: #architecture }
+
+The Substrate framework is designed for maximum customizability, providing a fully functional implementation for every important internal aspect of a blockchain. It allows developers to focus on the specifics of the use case and the runtime characteristics and to change any of the default features (should the need arise).
+
+The architecture of a Substrate node contains two main components:
+
+- **Core Client** - handles the communication with the outer world (other nodes, DApps, etc.), and many other internal responsibilities, such as storage and communication
+- **Runtime** - implements the custom logic of the Appchain, executes transactions, and manages the state transitions
+
+The end users can interact with the Appchain using the DApps (or directly via the node RPC endpoints) and get data or send transactions, which will remain queued until execution in the runtime.
```

Comment:
```suggestion
End users can interact with an Appchain using DApps (or directly via node RPC endpoints) and get data or send transactions, which will remain queued until execution in the runtime.
```

Haha I reread this section a couple times, and I just had the feeling that this sentence seemed out of place. I understand the point you're making here, but I just felt like it didn't quite fit for some reason. Maybe a smoother transition is needed? I don't know - perhaps reread it and let me know what you think?

Reply:
Yeah! It was subtly off somehow. I did rewrite the paragraph, emphasizing the user perspective. wdyt ?

Reply:
Yeah it looks great!